backtrace-library/src/test/java/backtraceio/library/TestUtils.java

```diff
@@ -0,0 +1,64 @@
+package backtraceio.library;
+
+import com.google.common.collect.MapDifference;
+import com.google.common.collect.Maps;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.lang.reflect.Type;
+import java.util.Map;
+
+public class TestUtils {
+
+    public static String readFileAsString(Object obj, String fileName) {
+        ClassLoader classLoader = obj.getClass().getClassLoader();
+        InputStream inputStream = classLoader.getResourceAsStream(fileName);
+
+        if (inputStream != null) {
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream))) {
+                StringBuilder jsonStringBuilder = new StringBuilder();
+                String line;
+                while ((line = reader.readLine()) != null) {
+                    jsonStringBuilder.append(line);
+                }
+                return jsonStringBuilder.toString();
+
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+        return null;
+    }
+
+    public static String unifyJsonString(String json) {
+        return json.replace("\n", "")
+                .replace(" ", "")
+                .replace("\t", "");
```

Comment:
This is invalid, this will replace characters inside quotes.
For example:
```
{
  "a": "a string"
}
```
would be changed to:
```
{"a":"astring"}
```

I'm not proficient in Java, but doesn't `Gson` have some methods to do that for you?

Reply:
good point, resolved